### DISCUSSION TITLE
Questions regarding statuses of notices and how the endpoints work

### DISCUSSION THREAD

**Discussion by EmilyBrasa** [OP-TED/eForms-SDK#219] [Q&A] [answered]
**Notice Author Email:** The API V2/notice/notices/submit demands an email: noticeAuthorEmail": "[john@doe.com](mailto:john@doe.com)". Whos email should this be? Should it be extracted from inside the eForms (which field)? Could multiple emails be given? What is this email used for? Is this the email that would be used to contact the provider if a rejection due to lawfulness manual check happens (eForms FAQ Question 12 https://docs.ted.europa.eu/home/eforms/FAQ/index.html#_forms_and_procedures)?
**Validation failed for warnings:** Currently when a validation report contains a lawfullness warning the status of the notice is set to VALIDATION_FAILED. Why does TED fail the validation even though there were no errors, just warnings? In the FAQ (https://docs.ted.europa.eu/eforms/latest/reference/business-rules/index.html) only 5 warnings are listed. Are only these warnings treated like errors?
**TED not_published after national publishing**: 48 hours after TED has changed the status of a notice to SUBMITTED eSenders are allowed to publish a notice nationally, even if TED has not published the notice yet. If after that time period TED returns status NOT_PUBLISHED because of a failed manual lawfulness check, we expect that TED will directly contact the procurement system via email and coordinate that the procurement system users will send a new corrected version of the notice. Is this expectation correct? In case a notice was already forwarded to a national procurement plattform do we have to automatically cancel the notice on the national level? How long could it take for TED to make a decision whether a notice is PUBLISHED or NOT_PUBLISHED?
**Notice Statusflow:** The status of a notice are described differently in the eForms FAQ ( https://docs.ted.europa.eu/home/eforms/FAQ/index.html), the TED eSender FAQ ( https://webgate.ec.europa.eu/fpfis/wikis/display/TEDeSender/2.5.1.+Notice+status) and in the eNotice2 API ( https://enotices2.ted.europa.eu/esenders/webjars/swagger-ui/index.html#). Why are there different statuses?
In the API the following status can be returned: DRAFT, SUBMITTED, STOPPED, PUBLISHING, PUBLISHED, DELETED, NOT_PUBLISHED, CONVERTED_FROM_PUBLISHED, ARCHIVED, VALIDATION_FAILED. What do these mean, especially DRAFT, PUBLISHING and CONVERTED_FROM_PUBLISHED?
**eForms Version for environments**: The preview environment currently only accepts eForms in version 1.3.2. However it was stated earlier that version 1.0.0 should always be accepted. We tried submitting an xml in Version 1.0 and got a 500 error. Which version of eForms is needed for the notice.xml in the future on production?

**Comment by rouschr** [comment, marked as answer]
I do not know the answers but I think some of this has been answered in the seminar today, mostly in the presentation of Karl "The lifecycle of eForms notices":
https://op.europa.eu/en/web/ted-eforms/agenda

**Reply by EmilyBrasa** [reply]
Yes, these questions are pretty old, most of them where answered already. Thanks anyway :)

**Reply by rouschr** [reply]
This will also be useful for Google searches landing here later :)
